src/main/scala/com/softwaremill/realworld/package.scala

```diff
@@ -0,0 +1,16 @@
+package com.softwaremill
+
+import com.softwaremill.realworld.articles.ArticlesEndpoints
+import sttp.tapir.swagger.bundle.SwaggerInterpreter
+import sttp.tapir.ztapir.ZServerEndpoint
+import zio.Task
+
+package object realworld {
+
+  private val apiEndpoints: List[ZServerEndpoint[Any, Any]] = ArticlesEndpoints.endpoints
```

Comment:
I don't think a package object is an intuitive place to look for these definitions; I'd move them to a normal object